lib/tasks/futurism_tasks.rake

```diff
@@ -34,6 +34,5 @@ namespace :futurism do
     subscribe_line = lines.find { |line| line.start_with?("Futurism.createSubscription") }
     lines << "Futurism.createSubscription(consumer)\n" unless subscribe_line
 
-    File.open(filepath, "w") { |f| f.write lines.join }
```

Comment:
why is this removed? did I miss sthg?

Reply:
Likely a fast-forward merge bug. I'll look upstairs